makefiles/boards/alias.inc.mk

```diff
@@ -0,0 +1,9 @@
+ALIAS = linux phynode
+
+# this basically implements a look-up table, is there a nicer way in make?
```

Comment:
the only other way that I know of is using variables inside variable names, but it's ugly in a different way.

```make
BOARD_ALIAS_linux = native
BOARD_ALIAS_phynode = pba-d-01-kw2x
ifneq (,$(BOARD_ALIAS_$(BOARD)))
  override BOARD = $(BOARD_ALIAS_$(BOARD))
endif
```